CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Release 0.25.12
+
+## What's New
+
```

Comment:
Can you add a note, something like: 'No functional changes, build process changes only'?

Reply:
Sure, I can expound upon

> CI updated